dev/update-output-repos.js

```diff
@@ -57,12 +57,8 @@ async function updateRepo(repoName) {
 }
 
 async function main() {
-  try {
```

Comment:
This change may be of importance.
I thought it ok to remove the try-catch, because running the script will print the error anyway.
Back when invoking this script via release-it, it probably made sense to swallow the error so release-it didn't fail.
But since the intent is to allow automation: https://github.com/ember-cli/ember-cli/pull/10211/files#diff-884b451b7291de286406620874239b6f24260847ef0b28d43beaa0b873d7032eR37

we no longer need to swallow the errors.